catroid/src/main/res/values/strings.xml

```diff
@@ -1076,14 +1074,13 @@
     <!--  -->
 
     <!-- Physics -->
-    <string name="default_project_name_physics">My first physics program</string>
     <string name="brick_set_physics_object_type">Set motion type to</string>
-    <string name="brick_set_physics_object_type_dynamic">bouncing with gravity</string>
-    <string name="brick_set_physics_object_type_fixed">others bounce off it</string>
-    <string name="brick_set_physics_object_type_no">no bouncing</string>
+    <string name="brick_set_physics_object_type_dynamic">moving and bouncing under gravity</string>
+    <string name="brick_set_physics_object_type_fixed">not moving under gravity, but others bounce off you under gravity</string>
+    <string name="brick_set_physics_object_type_no">not moving or bouncing under gravity (default)</string>
```

Comment:
These changes should also help in making it much clearer what to expect from these settings related to the physics engine.